_translations/en.yml

```diff
@@ -543,16 +543,16 @@
     pagedesc: "Using Bitcoin to pay and get paid is easy and accessible to everyone."
     users: "How to use Bitcoin"
     inform: "Inform yourself"
-    informtxt: "Bitcoin is different than what you know and use every day. Before you start using Bitcoin, there are a few things that you need to know in order to use it securely and avoid common pitfalls."
+    informtxt: "Bitcoin is different than what you know and use every day to make payments. Before you start using Bitcoin, there are a few things that you need to know in order to use it securely and avoid common pitfalls."
```

Comment:
Let's strike this change, because we can't assume that the context in which people are using bitcoin each day, is only to make payments.